### PR TITLE
[4.3.0] Update global key manager docs

### PR DESCRIPTION
**Pull request by SavinduDimal**
## Purpose

- Fix https://github.com/wso2/docs-apim/issues/7820

### CODE REVIEW

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 1]
```diff
@@ -59,23 +62,26 @@ Let's look at a scenario where a single access token generated for an applicatio
...
+7. Go to **Portal Configurations** from the left menu and click **Subscriptions**, click the **Subscription Availability** dropdown, and select the desired subscription availability option. For this scenario, select **Available to all the tenants** and Click Save.
 
-7. [Deploy the API]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api/)
+    [![Relationship between the API's visibility and subscription availability]({{base_path}}/assets/img/learn/api-subscription-availability.png)]({{base_path}}/assets/img/learn/api-subscription-availability.png)
 
-8. [Create a tenant]({{base_path}}/administer/multitenancy/managing-tenants/) (Ex: abc.com)
+8. [Deploy the API]({{base_path}}/deploy-and-publish/deploy-on-gateway/deploy-api/deploy-an-api/)
 
-9. Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the new tenant’s (abc.com) admin credentials and repeat Steps 5 - 7
+9.  Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the new tenant’s (abc.com) admin credentials and repeat Steps 5 - 7
```
Comment (hisanhunais) [inline en/docs/administer/key-managers/configure-global-key-manager.md:82]: Remove space

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 2]
```diff
@@ -59,23 +62,26 @@ Let's look at a scenario where a single access token generated for an applicatio
...
 
-10. Sign in to the WSO2 API Manager Developer Portal using the super tenant’s admin credentials (`https://<hostname>:9443/devportal`)
+10. Sign in to the super tenant's WSO2 API Manager Developer Portal using the super tenant’s admin credentials (`https://<hostname>:9443/devportal`)
```
Comment (hisanhunais) [inline en/docs/administer/key-managers/configure-global-key-manager.md:84]: Lets make this super tenant's Developer Portal

File: en/docs/administer/key-managers/configure-global-key-manager.md [hunk 3]
```diff
@@ -59,23 +62,26 @@ Let's look at a scenario where a single access token generated for an applicatio
...
+
+5. Sign in to the WSO2 API Publisher (`https://<hostname>:9443/publisher`) using the super tenant’s admin credentials
 
-5. [Create an API]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) (Eg: SampleAPI)
+6. [Create an API]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) (Eg: SampleAPI)
 
     [![Sample API Creation]({{base_path}}/assets/img/administer/global-keymanager/sample-api-creation.png)]({{base_path}}/assets/img/administer/global-keymanager/sample-api-creation.png)
 
-6. Go to **Portal Configurations** from the left menu and click **Subscriptions**, click the **Subscription Availability** dropdown, and select the desired subscription availability option. For this scenario, select **Available to all the tenants** and Click Save.
+    !!! Note
```
Comment (hisanhunais) [inline en/docs/administer/key-managers/configure-global-key-manager.md:73]: Lets add this note after step 7

**Review by hisanhunais — APPROVED** [review]

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 2 above)
Comment (SavinduDimal) [reply en/docs/administer/key-managers/configure-global-key-manager.md]: Resolved by https://github.com/wso2/docs-apim/pull/7921/commits/a66b668f482b7cb1d1c03f2e2df6bbf92ab0f1ed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 3 above)
Comment (SavinduDimal) [reply en/docs/administer/key-managers/configure-global-key-manager.md]: Resolved by https://github.com/wso2/docs-apim/pull/7921/commits/a66b668f482b7cb1d1c03f2e2df6bbf92ab0f1ed

File: en/docs/administer/key-managers/configure-global-key-manager.md (hunk 1 above)
Comment (SavinduDimal) [reply en/docs/administer/key-managers/configure-global-key-manager.md]: Resolved by https://github.com/wso2/docs-apim/pull/7921/commits/a66b668f482b7cb1d1c03f2e2df6bbf92ab0f1ed

**Pull request merged**